### DISCUSSION TITLE
Error while loading shared libraries: libprotobuf.so.17: no such file or directory

### DISCUSSION THREAD

**Discussion by olddisk** [protobuf-c/protobuf-c#630] [Q&A] [answered]
[Title]

**Comment by Avin-Techv** [comment]
i too get this error when installing tensorflow via make command

this is the error i get for this command

tensorflow/contrib/makefile/gen/bin/benchmark  --graph=$HOME/graphs/inception/tensorflow_inception_graph.pb

tensorflow/contrib/makefile/gen/bin/benchmark: error while loading shared libraries: libprotobuf.so.17: cannot open shared object file: No such file or directory

**Comment by sathyarr** [comment]
[found same error](https://github.com/tensorflow/tensorflow/issues/25576). any updates?

**Comment by lipnitsk** [comment, marked as answer]
`libprotobuf.so` is exported by the protobuf library, not protobuf-c. Please contact libprotobuf developers if you are still seeing issues.
